pkg/virtualKubelet/provider/provider.go

```diff
@@ -120,6 +128,11 @@ func NewLiqoProvider(ctx context.Context, cfg *InitConfig) (*LiqoProvider, error
 		With(exposition.NewEndpointSliceReflector(forge.IPAMClient(), cfg.EndpointSliceWorkers)).
 		With(configuration.NewConfigMapReflector(cfg.ConfigMapWorkers)).
 		With(configuration.NewSecretReflector(cfg.SecretWorkers))
+	if cfg.EnableStorage {
+		reflectionManager = reflectionManager.With(storage.NewPersistentVolumeClaimReflector(cfg.PersistenVolumeClaimWorkers, cfg.RemoteClusterID,
+			cfg.VirtualStorageClassName, cfg.RemoteRealStorageClassName,
+			eb.NewRecorder(scheme.Scheme, corev1.EventSource{Component: path.Join("remote-storage-provisioner", cfg.RemoteClusterID)})))
```

Comment:
A couple of (minor) comments:
1. It is not necessary to assign the reflection manager variable, as the modification is done in-place
2. The remote cluster ID is already exposed as a public variable in the forge package, and does not need to be given as parameter
3. The event broadcaster should be added to the manager and exposed as option (as might be used by multiple reflectors)